modules/capacity_factors_wind_pv/config/default.yaml

```diff
@@ -0,0 +1,67 @@
+# Module input files by users go here.
+resources:
+  download: True
+  spatial_units: https://www.dropbox.com/scl/fi/sjjisuqnzck2mhxs9jca7/units-{resolution}.geojson?rlkey=rv0q1exsytxws3gfvankbt9p7&e=1&dl=0
+
+# Any other user configuration goes below
+data-sources:
+    eez: https://sandbox.zenodo.org/records/45135/files/eez_v11.gpkg.zip?download=1  # FIXME: update to actual zenodo record before next Euro-Calliope release.
+    capacity-factors: https://zenodo.org/record/3899687/files/{filename}?download=1
+resolution: "ehighways"
+scope:
+  spatial:
+      countries:
+          - "Austria"
+          - "Belgium"
+          - "Bulgaria"
+          - "Croatia"
+          - "Cyprus"
+          - "Czech Republic"
+          - "Denmark"
+          - "Estonia"
+          - "Finland"
+          - "France"
+          - "Germany"
+          - "Greece"
+          - "Hungary"
+          - "Ireland"
+          - "Italy"
+          - "Latvia"
+          - "Lithuania"
+          - "Luxembourg"
+          - "Netherlands"
+          - "Poland"
+          - "Portugal"
+          - "Romania"
+          - "Slovakia"
+          - "Slovenia"
+          - "Spain"
+          - "Sweden"
+          - "United Kingdom"
+          - "Albania"
+          - "Bosnia and Herzegovina"
+          - "Macedonia, Republic of"
+          - "Montenegro"
+          - "Norway"
+          - "Serbia"
+          - "Switzerland"
+      bounds:
+          x_min: -15.8
+          x_max: 37
+          y_min: 30
+          y_max: 75
+  temporal:
+      first-year: 2016
+      final-year: 2016
```

Comment:
To make the workflow much more flexible, try changing things to use a wildcard (`{resolution}/{year}`), removing the need for this configuration.
It might be necessary to check your sourced datasets for the years they enable, and set a global wildcard constraint for it.

```
wildcard_constraint:
    year = "|".join([str(i) for i in range(2000, 2019)])
```

Reply:
Will harmonize that soon, also in the other new modules, but first I'll merge.

Reply:
Agreed. This can wait for later.